src/main/java/graphql/execution/ExecutionStrategy.java

```diff
@@ -140,12 +140,7 @@ protected ExecutionResult completeValue(ExecutionContext executionContext, Execu
             resolvedType = (GraphQLObjectType) fieldType;
         }
 
-        Map<String, List<Field>> subFields = new LinkedHashMap<>();
```

Comment:
Notice how visitedFragments is NEVER used.  Thats why I took it out of the public method signature